Sources/NIOHTTP2/HTTP2FrameParser.swift

```diff
@@ -601,6 +619,11 @@ struct HTTP2FrameDecoder {
                 throw NIOHTTP2Errors.excessivelyLargeHeaderBlock()
             }
 
+            // The sequence of CONTINUATION frames received is not longer than the configured limit
+            guard self.sequentialContinuationFramesCount < maximumSequentialContinuationFrames else {
```

Comment:
Shouldn't this be `<=`?

Reply:
No, it shouldn't be. But I have made some adjustments to make the condition clearer.